pom.xml

```diff
@@ -734,6 +755,18 @@
                             <targetPath>default-site/templates/alpha</targetPath>
                         </resource>
 
+                        <!-- video center blueprint -->
+                        <resource>
+                            <directory>${studio.vcbp.path}</directory>
+                            <filtering>false</filtering>
+                            <targetPath>repo-bootstrap/global/blueprints/video-center-blueprint</targetPath>
```

Comment:
Please rename to `video-center` (no `blueprint`) since that's the parent folder name anyway.